tests/test-async.el

```diff
@@ -1,4 +1,5 @@
 ;; -*- lexical-binding: t -*-
+(require 'subr-x)
```

Comment:
If this is just for `string-join` then it can be wrapped in `eval-when-compile`.

Reply:
I don't think the tests get compiled though.